app/scrapers/baidu.py

```diff
@@ -10,7 +10,7 @@ def __init__(self):
         self.url = 'https://www.baidu.com/s'
         self.newsURL = 'http://news.baidu.com/ns'
         self.defaultStart = 0
-        self.queryKey = 'wd'
+        self.queryKey = 'word'
```

Comment:
queryKey = 'wd' is for General search and queryKey = 'word' is for news so both required at same time

Reply:
No @bhaveshAn both 'wd' and 'word' are applicable. http://www.baidu.com/s?wd=hello and http://www.baidu.com/s?word=hello both give the same results. For general search, both wd and word are fine but for news search, word is needed. That's why I've changed the queryKey to word.
You can check my Heroku URL if you want further proof that it works 😄